networking/ingress/getting-traffic-cluster.adoc

```diff
@@ -0,0 +1,101 @@
+[id="getting-traffic-cluster"]
+= Getting traffic into a cluster
+include::modules/common-attributes.adoc[]
+:context: getting-traffic-cluster
+
+toc::[]
+
+{product-title} provides multiple methods for communicating from
+outside the cluster with services running in the cluster.
+
+[NOTE]
+====
+The procedures in this section require prerequisites performed by the cluster
+administrator.
+====
+
+Administrators can expose a service endpoint that external traffic can reach, by
+assigning a unique external IP address to that service from a range of external
+IP addresses. Administrators can designate a range of addresses using a CIDR
+notation, which allows an application user to make a request against the cluster
+for an external IP address.
+
+Each IP address should be assigned to only one service to ensure that each
+service has a unique endpoint. Potential port clashes are handled on a
+first-come, first-served basis.
+
+The recommendation, in order or preference, is:
+
+* If you have HTTP/HTTPS, use an ingress controller.
+* If you have a TLS-encrypted protocol other than HTTPS (for example, TLS with
+ the SNI header), use an ingress controller.
+* Otherwise, use a Load Balancer, an External IP, or a NodePort.
+
+[[external-access-options-table]]
+[options="header"]
+|===
+
+|Method |Purpose
+
+|Use an ingress controller
+|Allows access to HTTP/HTTPS traffic and TLS-encrypted protocols other than HTTPS (for example, TLS with the SNI header).
+
+|Automatically Assign a Public IP Using a Load Balancer Service
+|Allows traffic to non-standard ports through an IP address assigned from a pool.
```

Comment:
Note: For now this is accurate, but we plan to allow private load balancers the future.